<?xml version='1.0' encoding='UTF-8'?>
<testsuites tests="72" failures="27" errors="0" skipped="0" time="65.401"><testsuite name="Root Suite" timestamp="2023-08-15T14:36:29" tests="0" file="cypress/e2e/Ventes/Clients.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Clients" timestamp="2023-08-15T14:36:29" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Clients &quot;before all&quot; hook for &quot;Get_ExportClients_Excel Separtor COMMA 200&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get_ExportClients_Excel Separtor COMMA 200&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Clients`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Clients`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:35:17" tests="0" file="cypress/e2e/Achats/SuppliersMetrics.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Suppliers Metrics" timestamp="2023-08-15T14:35:17" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Suppliers Metrics &quot;before all&quot; hook for &quot;Get total amount of invoices associated to a supplier by exercise month&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get total amount of invoices associated to a supplier by exercise month&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Suppliers Metrics`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Suppliers Metrics`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:55:21" tests="0" file="cypress/e2e/Achats/Suppliers.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Suppliers" timestamp="2023-08-15T13:55:21" tests="5" time="5.562" failures="0" errors="0" skipped="0">
    <testcase name="Suppliers Get Suppliers " time="2.298" classname="Get Suppliers ">
    </testcase>
    <testcase name="Suppliers Get Export a Supplier " time="0.310" classname="Get Export a Supplier ">
    </testcase>
    <testcase name="Suppliers Post Supplier 201" time="1.009" classname="Post Supplier 201">
    </testcase>
    <testcase name="Suppliers Get a Supplier List" time="0.948" classname="Get a Supplier List">
    </testcase>
    <testcase name="Suppliers Delete Supplier 201" time="0.997" classname="Delete Supplier 201">
    </testcase>
  </testsuite>
  <testsuite name="Rules" timestamp="2023-08-15T13:55:27" tests="9" time="5.816" failures="0" errors="0" skipped="0">
    <testcase name="Suppliers Rules Post Supplier 400 France" time="0.313" classname="Post Supplier 400 France">
    </testcase>
    <testcase name="Suppliers Rules Post Supplier 201 France" time="1.044" classname="Post Supplier 201 France">
    </testcase>
    <testcase name="Suppliers Rules Get data about the siret" time="0.316" classname="Get data about the siret">
    </testcase>
    <testcase name="Suppliers Rules Post Supplier 201 With contacts" time="1.192" classname="Post Supplier 201 With contacts">
    </testcase>
    <testcase name="Suppliers Rules Post Supplier 201 With paymentInfo" time="1.428" classname="Post Supplier 201 With paymentInfo">
    </testcase>
    <testcase name="Suppliers Rules Post Supplier 403 Invalid Token" time="0.304" classname="Post Supplier 403 Invalid Token">
    </testcase>
    <testcase name="Suppliers Rules Post Supplier 401 Not Authorized" time="0.410" classname="Post Supplier 401 Not Authorized">
    </testcase>
    <testcase name="Suppliers Rules Post Supplier 404 Not Found" time="0.406" classname="Post Supplier 404 Not Found">
    </testcase>
    <testcase name="Suppliers Rules Post Supplier 400 Malformaed" time="0.403" classname="Post Supplier 400 Malformaed">
    </testcase>
  </testsuite>
  <testsuite name="Supplier Notes" timestamp="2023-08-15T13:55:32" tests="1" time="2.363" failures="0" errors="0" skipped="0">
    <testcase name="Supplier Notes Get Supplier Families List Notes" time="2.363" classname="Get Supplier Families List Notes">
    </testcase>
  </testsuite>
  <testsuite name="OCR" timestamp="2023-08-15T13:55:35" tests="1" time="4.496" failures="0" errors="0" skipped="0">
    <testcase name="Supplier Notes OCR Get documents references list 404" time="4.496" classname="Get documents references list 404">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:37:41" tests="0" file="cypress/e2e/Ventes/DownloadImportModel.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="download-import-model" timestamp="2023-08-15T14:37:41" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="download-import-model &quot;before all&quot; hook for &quot;Get download-import-model CLIENTS&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get download-import-model CLIENTS&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `download-import-model`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `download-import-model`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:32:21" tests="0" file="cypress/e2e/Achats/PurchasesPayment.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Purchases Payment" timestamp="2023-08-15T14:32:21" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Purchases Payment &quot;before all&quot; hook for &quot;Get a list of payments&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get a list of payments&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Purchases Payment`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Purchases Payment`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:57:59" tests="0" file="cypress/e2e/Ventes/Countries.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Countries" timestamp="2023-08-15T13:57:59" tests="1" time="3.133" failures="0" errors="0" skipped="0">
    <testcase name="Countries Get Countries" time="3.133" classname="Get Countries">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:58:17" tests="0" file="cypress/e2e/Ventes/TaskManagement.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Sales" timestamp="2023-08-15T13:58:17" tests="1" time="4.57" failures="0" errors="0" skipped="0">
    <testcase name="Sales Get Task Management" time="4.570" classname="Get Task Management">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:38:53" tests="0" file="cypress/e2e/Ventes/TaskManagement.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Sales" timestamp="2023-08-15T14:38:53" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Sales &quot;before all&quot; hook for &quot;Get Task Management&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get Task Management&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Sales`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Sales`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:55:06" tests="0" file="cypress/e2e/Achats/SiretDataBase.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Suppliers" timestamp="2023-08-15T13:55:06" tests="1" time="2.325" failures="0" errors="0" skipped="0">
    <testcase name="Suppliers Get data about the siret" time="2.325" classname="Get data about the siret">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:33:32" tests="0" file="cypress/e2e/Achats/SupplierFamily.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Suppliers Family" timestamp="2023-08-15T14:33:32" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Suppliers Family &quot;before all&quot; hook for &quot;Post Supplier Family 201&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Post Supplier Family 201&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Suppliers Family`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Suppliers Family`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:31:45" tests="0" file="cypress/e2e/Achats/Invoices.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Invoices" timestamp="2023-08-15T14:31:45" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Invoices &quot;before all&quot; hook for &quot;Get a list of invoices payments&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get a list of invoices payments&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Invoices`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Invoices`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:32:57" tests="0" file="cypress/e2e/Achats/SiretDataBase.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Suppliers" timestamp="2023-08-15T14:32:57" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Suppliers &quot;before all&quot; hook for &quot;Get data about the siret&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get data about the siret&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Suppliers`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Suppliers`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:58:06" tests="0" file="cypress/e2e/Ventes/DownloadImportModel.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="download-import-model" timestamp="2023-08-15T13:58:06" tests="2" time="2.781" failures="0" errors="0" skipped="0">
    <testcase name="download-import-model Get download-import-model CLIENTS" time="2.384" classname="Get download-import-model CLIENTS">
    </testcase>
    <testcase name="download-import-model Get download-import-model ARTICLES" time="0.397" classname="Get download-import-model ARTICLES">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:54:51" tests="0" file="cypress/e2e/Achats/Invoices.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Invoices" timestamp="2023-08-15T13:54:51" tests="2" time="0.313" failures="1" errors="0" skipped="0">
    <testcase name="Invoices Get a list of invoices payments" time="0.000" classname="Get a list of invoices payments">
      <failure message="expected 404 to equal 200" type="AssertionError">AssertionError: expected 404 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/Invoices.cy.js:43:35)

      + expected - actual

      -404
      +200
      </failure>
    </testcase>
    <testcase name="Invoices Get Invoices Payments Summary" time="0.313" classname="Get Invoices Payments Summary">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:55:12" tests="0" file="cypress/e2e/Achats/SupplierFamily.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Suppliers Family" timestamp="2023-08-15T13:55:12" tests="5" time="4.612" failures="2" errors="0" skipped="0">
    <testcase name="Suppliers Family Post Supplier Family 201" time="3.284" classname="Post Supplier Family 201">
    </testcase>
    <testcase name="Suppliers Family Get Supplier Family  200" time="0.000" classname="Get Supplier Family  200">
      <failure message="expected 405 to equal 200" type="AssertionError">AssertionError: expected 405 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/SupplierFamily.cy.js:52:43)

      + expected - actual

      -405
      +200
      </failure>
    </testcase>
    <testcase name="Suppliers Family Get List Supplier Family 200" time="0.352" classname="Get List Supplier Family 200">
    </testcase>
    <testcase name="Suppliers Family Get Export Supplier Family 200" time="0.000" classname="Get Export Supplier Family 200">
      <failure message="expected 405 to equal 200" type="AssertionError">AssertionError: expected 405 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/SupplierFamily.cy.js:80:39)

      + expected - actual

      -405
      +200
      </failure>
    </testcase>
    <testcase name="Suppliers Family Delete  Supplier Family 200" time="0.976" classname="Delete  Supplier Family 200">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:40:04" tests="0" file="cypress/e2e/Ventes/Winsis_Settings.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Sales" timestamp="2023-08-15T14:40:04" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Sales &quot;before all&quot; hook for &quot;Get Winsis Settings&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get Winsis Settings&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Sales`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Sales`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:58:25" tests="0" file="cypress/e2e/Ventes/ValueRange.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Countries" timestamp="2023-08-15T13:58:25" tests="7" time="6.371" failures="0" errors="0" skipped="0">
    <testcase name="Countries Get Value Range Quotes AMOUNT_TTC" time="3.134" classname="Get Value Range Quotes AMOUNT_TTC">
    </testcase>
    <testcase name="Countries Get Value Range INVOICES AMOUNT_TTC" time="1.016" classname="Get Value Range INVOICES AMOUNT_TTC">
    </testcase>
    <testcase name="Countries Get Value Range ARTICLES AMOUNT_TTC" time="1.020" classname="Get Value Range ARTICLES AMOUNT_TTC">
    </testcase>
    <testcase name="Countries Get Value Range CHECKS AMOUNT_TTC" time="0.307" classname="Get Value Range CHECKS AMOUNT_TTC">
    </testcase>
    <testcase name="Countries Get Value Range ARTICLE_FAMILIES ARTICLES_QTY" time="0.290" classname="Get Value Range ARTICLE_FAMILIES ARTICLES_QTY">
    </testcase>
    <testcase name="Countries Get Value Range CLIENT_FAMILIES ARTICLE CLIENTS_QTY" time="0.316" classname="Get Value Range CLIENT_FAMILIES ARTICLE CLIENTS_QTY">
    </testcase>
    <testcase name="Countries Get Value Range RECURRENCES  AMOUNT_TTC" time="0.288" classname="Get Value Range RECURRENCES  AMOUNT_TTC">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:37:05" tests="0" file="cypress/e2e/Ventes/Countries.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Countries" timestamp="2023-08-15T14:37:05" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Countries &quot;before all&quot; hook for &quot;Get Countries&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get Countries&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Countries`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Countries`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:34:08" tests="0" file="cypress/e2e/Achats/Suppliers.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Suppliers" timestamp="2023-08-15T14:34:08" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Suppliers &quot;before all&quot; hook for &quot;Get Suppliers &quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get Suppliers &quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Suppliers`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Suppliers`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
  <testsuite name="Supplier Notes" timestamp="2023-08-15T14:34:39" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Supplier Notes &quot;before all&quot; hook for &quot;Get Supplier Families List Notes&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get Supplier Families List Notes&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://purchase-qa.facilia.com/purchases/companies/2/suppliers  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Supplier Notes`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://purchase-qa.facilia.com/purchases/companies/2/suppliers

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Supplier Notes`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:38:17" tests="0" file="cypress/e2e/Ventes/Invoices_LastPaid.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Invoices Last Paid" timestamp="2023-08-15T14:38:17" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Invoices Last Paid &quot;before all&quot; hook for &quot;Get Invoices Last Paid&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get Invoices Last Paid&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Invoices Last Paid`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Invoices Last Paid`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:55:52" tests="0" file="cypress/e2e/Achats/Validations.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Validations" timestamp="2023-08-15T13:55:52" tests="5" time="5.667" failures="0" errors="0" skipped="0">
    <testcase name="Validations Invoices Validations Summary" time="2.279" classname="Invoices Validations Summary">
    </testcase>
    <testcase name="Validations Get a list of invoices validations" time="0.450" classname="Get a list of invoices validations">
    </testcase>
    <testcase name="Validations Export validations Excel" time="1.052" classname="Export validations Excel">
    </testcase>
    <testcase name="Validations Export validations CSV COMMA" time="0.941" classname="Export validations CSV COMMA">
    </testcase>
    <testcase name="Validations Export validations CSV SEMICOLON" time="0.945" classname="Export validations CSV SEMICOLON">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:58:35" tests="0" file="cypress/e2e/Ventes/Winsis_Settings.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Sales" timestamp="2023-08-15T13:58:35" tests="1" time="3.379" failures="0" errors="0" skipped="0">
    <testcase name="Sales Get Winsis Settings" time="3.379" classname="Get Winsis Settings">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:35:53" tests="0" file="cypress/e2e/Achats/Validations.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Validations" timestamp="2023-08-15T14:35:53" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Validations &quot;before all&quot; hook for &quot;Invoices Validations Summary&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Invoices Validations Summary&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Validations`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Validations`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:56:02" tests="0" file="cypress/e2e/Ventes/Clients.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Clients" timestamp="2023-08-15T13:56:02" tests="9" time="11.627" failures="3" errors="0" skipped="0">
    <testcase name="Clients Get_ExportClients_Excel Separtor COMMA 200" time="0.000" classname="Get_ExportClients_Excel Separtor COMMA 200">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: GET URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=CSV&amp;exportFile%5Bseparator%5D=COMMA  No response was received within the timeout.  https://on.cypress.io/request" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=CSV&amp;exportFile%5Bseparator%5D=COMMA

No response was received within the timeout.

https://on.cypress.io/request
    at &lt;unknown&gt; (http://localhost:46629/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:46629/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:46629/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:46629/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:46629/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:46629/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:46629/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:46629/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:46629/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:46629/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:46629/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:46629/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
    <testcase name="Clients Get_ExportClients_Excel 200" time="0.000" classname="Get_ExportClients_Excel 200">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: GET URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=EXCEL  No response was received within the timeout.  https://on.cypress.io/request" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=EXCEL

No response was received within the timeout.

https://on.cypress.io/request
    at &lt;unknown&gt; (http://localhost:46629/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:46629/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:46629/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:46629/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:46629/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:46629/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:46629/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:46629/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:46629/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:46629/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:46629/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:46629/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
    <testcase name="Clients Get_ExportClients_Excel exportFile 403" time="0.000" classname="Get_ExportClients_Excel exportFile 403">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: GET URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=CSV&amp;exportFile%5Bseparator%5D=  No response was received within the timeout.  https://on.cypress.io/request" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: GET
URL: https://recette-int-qa.facilia.com/companies/2/clients/export?companyId=2&amp;exportFile%5Btype%5D=CSV&amp;exportFile%5Bseparator%5D=

No response was received within the timeout.

https://on.cypress.io/request
    at &lt;unknown&gt; (http://localhost:46629/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:46629/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:46629/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:46629/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:46629/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:46629/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:46629/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:46629/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:46629/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:46629/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:46629/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:46629/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
    <testcase name="Clients Client registration France 201" time="4.724" classname="Client registration France 201">
    </testcase>
    <testcase name="Clients Client registration without_contact_email" time="0.818" classname="Client registration without_contact_email">
    </testcase>
    <testcase name="Clients Client registration without_contact_city" time="0.856" classname="Client registration without_contact_city">
    </testcase>
    <testcase name="Clients Client registration Clients_TOKEN_NOK" time="0.909" classname="Client registration Clients_TOKEN_NOK">
    </testcase>
    <testcase name="Clients Get_Client " time="3.239" classname="Get_Client ">
    </testcase>
    <testcase name="Clients Get_Client List " time="1.081" classname="Get_Client List ">
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:55:00" tests="0" file="cypress/e2e/Achats/PurchasesPayment.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Purchases Payment" timestamp="2023-08-15T13:55:00" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Purchases Payment Get a list of payments" time="0.000" classname="Get a list of payments">
      <failure message="expected 500 to equal 200" type="AssertionError">AssertionError: expected 500 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/PurchasesPayment.cy.js:39:35)

      + expected - actual

      -500
      +200
      </failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T14:39:28" tests="0" file="cypress/e2e/Ventes/ValueRange.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Countries" timestamp="2023-08-15T14:39:28" tests="1" time="0.0" failures="1" errors="0" skipped="0">
    <testcase name="Countries &quot;before all&quot; hook for &quot;Get Value Range Quotes AMOUNT_TTC&quot;" time="0.000" classname="&quot;before all&quot; hook for &quot;Get Value Range Quotes AMOUNT_TTC&quot;">
      <failure message="`cy.request()` timed out waiting `30000ms` for a response from your server.  The request we sent was:  Method: POST URL: https://recette-int-qa.facilia.com/login  No response was received within the timeout.  https://on.cypress.io/request  Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Countries`" type="CypressError">CypressError: `cy.request()` timed out waiting `30000ms` for a response from your server.

The request we sent was:

Method: POST
URL: https://recette-int-qa.facilia.com/login

No response was received within the timeout.

https://on.cypress.io/request

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Countries`
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:149368:78)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at &lt;unknown&gt; (http://localhost:36087/__cypress/runner/cypress_runner.js:13866:41)
    at tryCatcher (http://localhost:36087/__cypress/runner/cypress_runner.js:18744:23)
    at Promise._settlePromiseFromHandler (http://localhost:36087/__cypress/runner/cypress_runner.js:16679:31)
    at Promise._settlePromise (http://localhost:36087/__cypress/runner/cypress_runner.js:16736:18)
    at Promise._settlePromise0 (http://localhost:36087/__cypress/runner/cypress_runner.js:16781:10)
    at Promise._settlePromises (http://localhost:36087/__cypress/runner/cypress_runner.js:16857:18)
    at _drainQueueStep (http://localhost:36087/__cypress/runner/cypress_runner.js:13451:12)
    at _drainQueue (http://localhost:36087/__cypress/runner/cypress_runner.js:13444:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13460:5)
    at Async.drainQueues (http://localhost:36087/__cypress/runner/cypress_runner.js:13330:14)
From Your Spec Code:
    at Context.ee (webpack:///./node_modules/cypress-plugin-api/dist/support.js:14:2002)</failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:55:44" tests="0" file="cypress/e2e/Achats/SuppliersMetrics.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Suppliers Metrics" timestamp="2023-08-15T13:55:44" tests="5" time="0.0" failures="5" errors="0" skipped="0">
    <testcase name="Suppliers Metrics Get total amount of invoices associated to a supplier by exercise month" time="0.000" classname="Get total amount of invoices associated to a supplier by exercise month">
      <failure message="expected 404 to equal 200" type="AssertionError">AssertionError: expected 404 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/SuppliersMetrics.cy.js:39:35)

      + expected - actual

      -404
      +200
      </failure>
    </testcase>
    <testcase name="Suppliers Metrics Get total amount of invoices associated to a supplier by exercise year" time="0.000" classname="Get total amount of invoices associated to a supplier by exercise year">
      <failure message="expected 404 to equal 200" type="AssertionError">AssertionError: expected 404 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/SuppliersMetrics.cy.js:57:35)

      + expected - actual

      -404
      +200
      </failure>
    </testcase>
    <testcase name="Suppliers Metrics Get total amount of invoices associated to a supplier by civil year (monthly)" time="0.000" classname="Get total amount of invoices associated to a supplier by civil year (monthly)">
      <failure message="expected 404 to equal 200" type="AssertionError">AssertionError: expected 404 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/SuppliersMetrics.cy.js:71:35)

      + expected - actual

      -404
      +200
      </failure>
    </testcase>
    <testcase name="Suppliers Metrics Get total amount of invoices associated to a supplier by civil year (January 1st to December 31st)" time="0.000" classname="Get total amount of invoices associated to a supplier by civil year (January 1st to December 31st)">
      <failure message="expected 404 to equal 200" type="AssertionError">AssertionError: expected 404 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/SuppliersMetrics.cy.js:83:35)

      + expected - actual

      -404
      +200
      </failure>
    </testcase>
    <testcase name="Suppliers Metrics Get total amount of invoices of the suppliers for the last 12 months" time="0.000" classname="Get total amount of invoices of the suppliers for the last 12 months">
      <failure message="expected 404 to equal 200" type="AssertionError">AssertionError: expected 404 to equal 200
    at Context.eval (webpack:///./cypress/e2e/Achats/SuppliersMetrics.cy.js:96:35)

      + expected - actual

      -404
      +200
      </failure>
    </testcase>
  </testsuite>
<testsuite name="Root Suite" timestamp="2023-08-15T13:58:12" tests="0" file="cypress/e2e/Ventes/Invoices_LastPaid.cy.js" time="0" failures="0" errors="0" skipped="0">
  </testsuite>
  <testsuite name="Invoices Last Paid" timestamp="2023-08-15T13:58:12" tests="1" time="2.386" failures="0" errors="0" skipped="0">
    <testcase name="Invoices Last Paid Get Invoices Last Paid" time="2.386" classname="Get Invoices Last Paid">
    </testcase>
  </testsuite>
</testsuites>